{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -6,25 +6,15 @@ skipsdist = True
 
 [testenv]
 setenv =
-    DJANGO_SETTINGS_MODULE={{ cookiecutter.project_slug }}.settings.base
+    DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.config.settings.test
     PYTHONPATH={toxinidir}
-    SECRET_KEY=test
```

Comment:
I think we should either set all variables via the `tox.ini` file, or via the `settings/test.py` file. Currently it's not possible to set them all via `settings/test.py` because `DATABASE_URL` is a required variable, but maybe we should just refactor this. We could have `base.py` and then `prod.py`, `test.py` and `dev.py`, and `DATABASE_URL` would be set in specific settings files (`prod.py`, `test.py`, etc), instead of in the base settings file. What do you think?